crates/modelardb_server/src/storage/types.rs

```diff
@@ -60,63 +60,80 @@ impl MemoryPool {
         uncompressed_memory_in_bytes: usize,
         compressed_memory_in_bytes: usize,
     ) -> Self {
-        // unwrap() is safe as i64::MAX is 8192 PiB and the value is from ConfigurationManager.
         Self {
             wait_for_ingested_memory: Condvar::new(),
             remaining_ingested_memory_in_bytes: Mutex::new(
-                ingested_memory_in_bytes.try_into().unwrap(),
+                ingested_memory_in_bytes
+                    .try_into()
+                    .expect("ingested_memory_in_bytes should be less than 8192 PiB."),
```

Comment:
Maybe use a variable instead of repeating the same string.

Reply:
Fixed,